examples/managed-identity/kubernetes.json

```diff
@@ -0,0 +1,38 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "kubernetesConfig": {
+        "useManagedIdentity": true

```

Comment:
Do you anticipate this to remain a true/false type setting or would there be other kinds of identities that might be available to users in the future?